kv/db_test.go

```diff
@@ -130,14 +130,15 @@ func TestKVDBInternalMethods(t *testing.T) {
 		args  proto.Request
 		reply proto.Response
 	}{
-		{&proto.InternalRangeLookupRequest{}, &proto.InternalRangeLookupResponse{}},
-		{&proto.InternalGCRequest{}, &proto.InternalGCResponse{}},
-		{&proto.InternalHeartbeatTxnRequest{}, &proto.InternalHeartbeatTxnResponse{}},
-		{&proto.InternalPushTxnRequest{}, &proto.InternalPushTxnResponse{}},
-		{&proto.InternalResolveIntentRequest{}, &proto.InternalResolveIntentResponse{}},
-		{&proto.InternalResolveIntentRangeRequest{}, &proto.InternalResolveIntentRangeResponse{}},
-		{&proto.InternalMergeRequest{}, &proto.InternalMergeResponse{}},
-		{&proto.InternalTruncateLogRequest{}, &proto.InternalTruncateLogResponse{}},
+		{&proto.HeartbeatTxnRequest{}, &proto.HeartbeatTxnResponse{}},
```

Comment:
I think the distinction is just that these are not accessible over HTTP (and thus not available to the client). Not sure it's worth it to eliminate the distinction given the move to batches in the near future.